deploy/frontend.yml

```diff
@@ -1,12 +1,12 @@
 apiVersion: v1
 kind: Template
 metadata:
-  name: api-frontend
+  name: api-docs
```

Comment:
@BlakeHolifield @karelhala @psav @adamrdrew 

I might be wrong but I do remember that the name must end with `-frontend`. Did this change recently?

Reply:
No, but it it should match the `insights.appnam` in package.json to avoid problems.